app/graphql/types/query_type.rb

```diff
@@ -15,5 +19,10 @@ def get_users
     def get_user(id:)
       User.find(id)
     end
+
+    # Eras
+    def get_era(id:)
```

Comment:
I find writing the argument like this `id:` a bit strange. My instinct was to just have `get_era(id)` does anyone know if that is graphql convention/where it comes from?

Reply:
I'm not sure but I was feeling the same way...